cmd/podman/rm.go

```diff
@@ -80,6 +80,9 @@ func rmCmd(c *cliconfig.RmValues) error {
 			return err
 		}
 		if err != nil {
+			if errors.Cause(err) == libpod.ErrNoSuchCtr {
+				exitCode = 1
+			}
 			fmt.Println(err.Error())
```

Comment:
I'm just wondering if we should error out at this point and not try the deletions?  If it's OK to continue on and delete what we can, then LGTM.

Reply:
we document that we return 1 if a container is not found

Reply:
the desired behaviour is to keep going and output at the end.